blog/tests.py

```diff
@@ -118,3 +132,172 @@ def test_delete_blog(self):
         force_authenticate(request, user=user)
         response = view(request, slug=blog.slug)
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+    def test_update_blog(self):
+        """
+        Tests to update a Blog Post Successfully.
+        """
+        response = self.create_blog()
+        user = User.objects.get(username='test')
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
move this line inside the `_create_blog` method.

Reply:
done.